FinSearchUnified/Resources/views/frontend/search/fuzzy.tpl

```diff
@@ -2,15 +2,16 @@
 
 {block name="frontend_search_headline"}
     <h1 class="search--headline">
-        {if $finSmartDidYouMean == false || $finSmartDidYouMean.type == 'did-you-mean'}
-            {s name='SearchHeadline'}{/s}
+        {if $snippetType == 'query'}
+            {s namespace='frontend/search/query_info_message' name='frontend/search/query_info_message/query'}{/s}
```

Comment:
Like we do for smart did you mean, add the English translation in between of `{s}{/s}`. This way the translations work even if the snippets do not yet exist, in case the plugin hasn't been reinstalled.